tools/generate-legioncomponent-template.js

```diff
@@ -0,0 +1,58 @@
+require('colors');
```

Comment:
Where are you invoking this script from? IF we are going to have a separate script for legion components, then how do we call it? There should be a package.json change to go with this.

Reply:
there is a package.json change that invokes the script